Authenticator/Source/AppViewController.swift

```diff
@@ -104,7 +104,7 @@ extension AppViewController: AppPresenter {
 
         case .EditForm(let formViewModel):
             if case .EditForm = currentViewModel.modal,
-                let editController = modalNavController?.topViewController as? TokenFormViewController {
+                let editController = modalNavController?.topViewController as? TokenFormViewController<TokenEditForm> {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 119 characters (line_length)